user_modules/shared_testcases/testCasesForPolicyTableSnapshot.lua

```diff
@@ -0,0 +1,387 @@
+local testCasesForPolicyTableSnapshot = {}
+local mobile_session = require('mobile_session')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+
+testCasesForPolicyTableSnapshot.preloaded_elements = {}
+testCasesForPolicyTableSnapshot.pts_elements = {}
+testCasesForPolicyTableSnapshot.seconds_between_retries = {}
+testCasesForPolicyTableSnapshot.preloaded_pt = {}
+
+-- --local data_dictionary = {}
+
+local json_elements = {}
+
+local function extract_json(pathToFile)
```

Comment:
@istoimenova , please provide a comment with description of Function
what does it do?
what parameters for what purpose?
what is return value if  is?

The comment for all Functions
BTW, for future we need some kind of Doxygen for automatic generating documentation.